webrtc.html

```diff
@@ -9712,7 +9712,37 @@ <h2>Attributes</h2>
             <dt><dfn data-idl><code>canInsertDTMF</code></dfn> of type <span class=
             "idlAttrType"><a>boolean</a></span>, readonly</dt>
             <dd>
-              <p>Whether the <a>RTCDTMFSender</a> is capable of sending DTMF.</p>
+              <p>Whether the <a>RTCDTMFSender</a> <var>dtmfSender</var> is capable of
+              sending DTMF. On getting, the attribute MUST return the value of
+              <var>dtmfSender</var>'s <a>[[\CanInsertDTMF]]</a> slot. To set
+              <var>dtmfSender</var>'s <a>[[\CanInsertDTMF]]</a> slot, run the following steps
+              when <code><var>pc</var>.connectionstatechange</code> is fired:
+              <ol>
+                <li>Let <var>pc</var> be the <code><a>RTCPeerConnection</a></code> which
+                is sending DTMF.</li>
+                <li>Let <var>transceiver</var> be the <code><a>RTCRtpTransceiver</a></code>
+                used to send DTMF.</li>
```

Comment:
This should be "the transceiver to which the RTPSender is associated", since [[CanInsertDTMF]] is a sender attribute.

Reply:
Fixed.